### PR TITLE
Add NestJS docs

### PR DESCRIPTION
**Pull request by KnorpelSenf**
Adds docs for the NestJS plugin so that we can release 1.0 and announce it on the news channel.

### CODE REVIEW

File: site/docs/plugins/nestjs.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+```sh
+npm install @grammjys/nestjs
+```
+
+to install the plugin in your existing NestJS project.
+
+## Usage
+
+todo
```
Comment (KnorpelSenf) [inline site/docs/plugins/nestjs.md:22]: @NarHakobyan please add a short example of a basic usage of this plugin by editing this file. It does not have to be anything fancy, just a simple echo bot to get people started maybe. I think it is fair to require familiarity with NestJS from our readers, so they should already have a working NestJS project when they get to this step.

**Comment by KnorpelSenf** [comment]
Nobody seems to care about NestJS docs. We are not going to release a new version of the plugin until this changes.

**Comment by anotheri** [comment]
> Nobody seems to care about NestJS docs. We are not going to release a new version of the plugin until this changes.

@KnorpelSenf  any chance to to get the focus back to nestjs plugin? any help/contributions needed?

**Comment by anotheri** [comment]
@KnorpelSenf well, it's hard to say why... I think nestjs tends to overcomplicate fairly simple bots and most people just don't need nestjs to build a bot at all (and as you said that's might be also a valid decision). Although nestjs (because of its opinionated nature) simplifies a lot of other things and for example if you build microservice-based architecture it helps a lot to keep consistent tech stack and reusable nestjs modules, so for some people it might also bring a convenience and ease of implementation.

Anyways, I'm going to work a bit closer with this lib (actually, with more or less stable fork of it), but since it depends on dated grammy and (even worse) dated nestjs, it's more on my own risks to go to prod with this plugin/fork, therefore i'd like to get more understanding what's going on under the hood and hopefully bring some stability and up to date dependencies. Can't promise that i won't give up on it in a week or so :) but if i'll have something valuable results i'll let you know. And if you have some vision or scope of work for this integration (what's been done and what's left) let me know as well. Thanks.

**Comment by NarHakobyan** [comment]
Hi everyone, I've updated the NestJS lib many times, but it wasn't merged because the website page was missing.

**Comment by NarHakobyan** [comment]
Hey, yes v11 is the latest version, we have forked grammy to our internal repos and migrated to v11.

Why we forked: v10 was migrated but was not published to npm so we had a lot of issues using it.

**Comment by anotheri** [comment]
hey there guys, so can we collaborate together to actually get it done and released?  does the plan i've mentioned [here](https://github.com/grammyjs/website/pull/1016#issuecomment-3077519118) look good? 
@NarHakobyan could you push your latest version with nest 11 and grammy 1.37 support? if no, then i'll have to do upgrade everything on my own (although i'd prefer to don't spend time on something that's already done),
@KnorpelSenf is any particular requirements for the website docs?

**Comment by KnorpelSenf** [comment]
There have been no commits in over a year. There seem to be no working forks or third-party implementations. After years of trying, it's time to give up on an official integration. We just can't live up to our quality standards.

Various people have complained that we never communicate the fact that the integration is effectively abandoned. That is misleading. Consequently, I have now archived the repo.

I encourage everyone to build a solid integration outside of the @grammyjs organisation. I'd be happy to list it as as a third-party plugin.

Thanks everyone for your efforts so far.

**Comment by anotheri** [comment]
@KnorpelSenf You have ignored the questions in last 2 messages for some reasons although I wanted to contribute and help with the project and this PR?

**Comment by KnorpelSenf** [comment]
Amazing, looking forward to it! 🚀

**Pull request closed without merging**